cmd/deploy/remote.go

```diff
@@ -224,3 +226,19 @@ func getOriginalCWD(cwd, manifestPath string) string {
 	manifestPathDir := filepath.Dir(fmt.Sprintf("/%s", manifestPath))
 	return strings.TrimSuffix(cwd, manifestPathDir)
 }
+
+func getOktetoCLIVersion() string {
```

Comment:
Can we add a test for this one?

Reply:
done 21650ac162c7da8df3bb040f6e988ee8a5c17691